shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -840,8 +842,8 @@ - (void)swipeWithEvent:(NSEvent*)event {
 - (void)touchesBeganWithEvent:(NSEvent*)event {
   NSTouch* touch = event.allTouches.anyObject;
   if (touch != nil) {
-    if (_mouseState.system_scroll_inertia_active) {
-      // The trackpad has been touched and a scroll gesture is still sending inertia events.
+    if ((event.timestamp - _mouseState.last_scroll_momentum_changed_time) < 0.050) {
```

Comment:
What about making this 0.050 a constant and explain the mechanism (such as L523) there?